docs/DeveloperGuide.md

```diff
@@ -270,14 +270,55 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​                                                    | I want to …​                                                 | So that I can…​                                                                                                    |
+|----------|------------------------------------------------------------|--------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| `* * *`  | new user                                                   | see usage instructions                                       | refer to instructions when I forget how to use the App                                                             |
+| `* * *`  | user                                                       | add companies                                                |                                                                                                                    |
+| `* * *`  | user                                                       | add events related to a company                              | keep track of all of a company’s events                                                                            |
+| `* * *`  | user                                                       | add contact people from a company                            | easily look up their contact details                                                                               |
```

Comment:
This implies that contact people already exists in the company right?

Reply:
Should we have a "add contact people to a company" or is it already included in the "edit details of a company"

Reply:
I think it should be add contact people to a company, I'll change that real quick